pkgs/development/interpreters/ruby/gemconfig/default.nix

```diff
@@ -129,10 +143,19 @@ in
 
   xapian-ruby = attrs: {
     # use the system xapian
+    dontBuild = false;
     buildInputs = [ xapian pkgconfig zlib ];
     postPatch = ''
       cp ${./xapian-Rakefile} Rakefile
     '';
+    # for whatever reason, the xapian-ruby gem leaves out the gemspec
+    # from the archive, so if we've extracted the source from a gem package,
+    # re-use the gemspec from the metadata.
```

Comment:
I think it's the other way around. Gems are not supposed to package their own gemspec, it's only because they are careless that it happens. I would extract and force the gemspec to be the one that comes from the metadata.